hack/run-dev.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script sets up a go workspace locally and run the autoheal server.
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+# Get host plataform
+platform="$(os::build::host_platform)"
+
+# Get platform binary and extention
+binary="${OS_OUTPUT_BINPATH}/${platform}/autoheal"
+if [[ $platform == "windows/amd64" ]]; then
+	binary=("${binary}.exe")
```

Comment:
This is unnecessary - not only we don't really support that platform, it's also unlikely a Windows user will be running bash :)

other than that LGTM